docs/ruleset_functions/get_SWH_uses_associated_with_each_building_segment.md

```diff
@@ -0,0 +1,28 @@
+## get_SWH_uses_associated_with_each_building_segment
+
+Description: This function gets all the SWH uses connected to a building segment.  This function is primarily to encapsulate getting service water heating uses in one function so that if a change is made in the schema as to how service water heating use is specified, the RCT only needs to change in one place.   
+
+Inputs:
+- **RMD**
+- **building_segment_id** - the id of the building segment
+
+Returns:
+- **swh_uses**: A list containing the ids of all service water heating uses associated with a building segment  
+
+Function Call:
+- get_component_by_ID  
```

Comment:
no... I'm not sure which is correct, but my understanding is that the RCT team understood either way what's going on

Reply:
I'm sure they would understand. I think I am just advocating for cleanliness more now because I am also currently going through every single TCD description and making revisions to have consistent wording and formatting.

Reply:
consistency is definitely better when possible!